Makefile

```diff
@@ -149,6 +149,7 @@ DEFENABLEDEBUG := false
 DEFDISABLENESTINGCHECKS := false
 DEFMSIZE9P := 8192
 DEFHOTPLUGVFIOONROOTBUS := false
+DEFGUESTHOOKPATH :=
```

Comment:
If the default value is empty, please drop `DEFGUESTHOOKPATH`.

Reply:
I will use `/usr/share/oci/hooks`, if that's fine for you. The option is commented by default anyway.

Reply:
OK, I missed the part where it's used for generating default values in a Go file.
To untangle the two I can remove the template from the `toml` file and put `/usr/share/oci/hooks`, and drop this variable from the Makefile.  Is that what you prefer?

Reply:
Updated my PR with what I mentioned above, in case my description was not clear. Let me know if I should change it back.